.github/PULL_REQUEST_TEMPLATE/pull_request_template.md

```diff
@@ -0,0 +1,4 @@
+<!--
+PR Creation Checklist
+- [ ] Update Changelog
+-->
```

Comment:
I put it inside a comment to emphasise it's for contributor to decide what to do with it.